### ISSUE TITLE
Add legal size from @SienaSantos fork

### ISSUE THREAD

**Issue by brody2consult** [cognitom/paper-css#19]
I discovered <https://github.com/SienaSantos/paper-css> fork which adds legal size in <https://github.com/SienaSantos/paper-css/commit/780e7030763afa7fc092ab5ecd9bb51cf66bd35f>.

**Comment by cognitom** [comment]
@brodybits it makes sense. We could add these lines, I think.

```css
body.letter       .sheet { width: 216mm; height: 279mm }
body.legal        .sheet { width: 216mm; height: 356mm }

@media print {
  body.letter, body.legal    { width: 216mm }
}
```

**Comment by cognitom** [comment]
Landed on #21

**Comment by brody2consult** [comment]
On Nov 21, 2017 6:05 AM, "Tsutomu Kawamura" <notifications@github.com>
wrote:

Landed on #21 <https://github.com/cognitom/paper-css/pull/21>

Thanks!
